src/devices/Is31fl3730/DotMatrix10x7.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Represents an IS31FL3731 LED Matrix driver
+    /// </summary>
+    // Datasheet: https://cdn-shop.adafruit.com/product-files/3017/31FL3730.pdf
```

Comment:
those comments goes into a /// <remark> comment

Reply:
I typically leave datasheet comments for the code. That strikes me as the best model. Thoughts?

Reply:
Fair enough! You're right, it does not necessarily brings a lot of value. We can assume the person has anyway access to the datasheet of the sensor so they can cable it properly. And also, we've seen that those URL can change. So good for the code.